packages/vant/src/text-ellipsis/README.zh-CN.md

```diff
@@ -124,6 +124,47 @@ export default {
 };
 ```
 
+### 自定义操作内容
+
+通过插槽 `expand-text` `collapse-text` 自定义操作内容
+
+```html
+<van-text-ellipsis ref="textEllipsisRef" :content="text">
+  <template #expand-text>
+    <van-button size="small" type="primary" @click="toggle(true)"
+      >展开</van-button
+    >
+  </template>
+  <template #collapse-text>
+    <van-button size="small" type="primary" @click="toggle(false)"
+      >收起</van-button
+    >
+  </template>
+</van-text-ellipsis>
+```
+
+```js
+import { ref } from 'vue';
+import type { TextEllipsisInstance } from 'vant';
```

Comment:
the demo code uses js by default, so we do not need to import type here.

Maybe our document can support switching language in the future

Reply:
get~

Reply:
emm, I adjusted code, review again~
now, I adjusted `action` slot, instead `expand-text` and `collapse-text`